src/components/common/CreateColonyWizard/CreateColonyWizard.ts

```diff
@@ -1,16 +1,26 @@
-import { ComponentType } from 'react';
```

Comment:
The idea here is to show the user creation wizard before colony creation if user isn't registered, right? It might be easier just to hide the create colony button so there's only one option for unregistered users, but alternatively, there could be a way of achieving what you want without modifying the existing wizards too heavily:
On Create colony button click, if user is unregistered, store a flag that tells you they want to create colony (it could be in context or local storage). Then simply redirect them to create user wizard. Modify the user wizard submission handler to check for the flag, and if it's present, redirect them to creating colony. 
That way, create colony doesn't need to know anything about user creation

Reply:
I will remove from this PR completely. While I agree hiding it is easier from a development perspective, it removes value from the user experience, as a users goal may be to create a colony and not just a user account. Essentially, hiding it removes a strong call to action.

Reply:
Yeah, I thought so, hence my alternative suggestion. Are you still planning to implement it in a separate PR?

Reply:
Yes, I will put it in a different PR.